crates/string-offsets/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "string-offsets"
+authors = ["The blackbird team <support@github.com>"]
+version = "0.1.0"
+edition = "2021"
+description = "Offset calculator to convert between byte, char, and line offsets in a string."
+repository = "https://github.com/github/rust-gems"
+license = "MIT"
+keywords = ["unicode", "string", "offsets", "positions", "interoperability"]
```

Comment:
should we add utf8 utf16 runes characters, etc. to the list of keywords?

Reply:
We only get five, but I like those, and probably we don't need "string" or "offsets". Will reshuffle.

Reply:
(Also, I suspect these don't matter much.)